### ISSUE TITLE
selectAction title should be passed to UI automation for accessibility

### ISSUE THREAD

**Issue by matthidinger** [microsoft/AdaptiveCards#1298]
<table>
<tr>
<th>Release</th>
<th>Renderer status</th>
<th>Tasks status</th>
</tr>
<tr>
<td>Backlog
<td>🔄 .NET (#2426)<br/>🔄 Android (#2427)<br/>🔄 iOS (#2428)<br/>✔️ TS (#2429)<br/>✔️ UWP (#2117)</td>
<td>✔️ Shared (#2425)</td>
</tr>
</table>

```json
      "selectAction": {
        "type": "Action.OpenUrl",
        "title": "Resume",
        "url": "http://adaptivecards.io"
      }
```

Even though the `title` doesn't get rendered in the UI, we should pass it to the UI framework on each renderer so that the object holding the `selectAction` is properly interpreted by screen readers.

# Auto-generated task status

- [x] Shared
- [ ] .NET
- [ ] Android
- [ ] iOS
- [x] TS
- [x] UWP

**Comment by jonmill** [comment]
This should be fixed by now
